test/helpers/kubectl.go

```diff
@@ -2142,14 +2142,6 @@ func (kub *Kubectl) overwriteHelmOptions(options map[string]string) error {
 		}
 	}
 
-	// Temporary workaround until https://github.com/cilium/cilium/pull/11915
-	// has been merged. Otherwise, the BPF verifier rejects some programs on
-	// the 4.19 kernel
-	if os.Getenv("KERNEL") == "419" {
-		options = addIfNotOverwritten(options, "global.kubeProxyReplacement", "partial")
-		options = addIfNotOverwritten(options, "global.hostServices.enabled", "true")
-	}
```

Comment:
Could you also pull in 6b14badba830a4c335d3e65fbf596deb7a1d2bab, so that kube-proxy is disabled and the code below runs? That way, `kube-proxy-replacement` would be `strict` and the host firewall would be enabled.

Reply:
Yes, agree. One thing we need to think about next is that we won't have any kube-proxy based coverage aside from only 4.9 kernel in CI.

Reply:
Yep, we will even have some tests that won't run anywhere anymore. For example, https://github.com/cilium/cilium/blob/6beb6b3d5a48e552395bd62669d3b8ebcbb0c222/test/k8sT/DatapathConfiguration.go#L208-L214
I was planning to have a look at all tests once v1.8 is out to see if any need to be reenable in some way.

Reply:
Agree, somehow it would be good to perhaps run 4.19 with one job in kube-proxy free and another with kube-proxy mode to be on safe side.